packages/mobile/e2e/src/usecases/ExchangeCelo.js

```diff
@@ -14,7 +14,36 @@ const CELO_TO_SELL = 0.045
 const CELO_TO_BUY = +Math.random().toFixed(3)
 const CELO_TO_WITHDRAW = +Math.random().toFixed(3)
 const CELO_TO_SELL_MIN = 0.002
-// TODO Fetch Fees
+
+const scrollToTopOfFeed = async () => {
+  // Scroll to top of feed - scroll not possible if not all transactions loaded
+  try {
+    await waitFor(element(by.id('ExchangeScrollView')))
+      .toBeVisible()
+      .withTimeout(10 * 1000)
+    await element(by.id('ExchangeScrollView')).scroll(200, 'down')
+  } catch {}
```

Comment:
the blind `catch` is a little worrisome.  Instead, can you check explicitly for the error you're expecting?

Reply:
This is to only attempt to scroll if we can. `waitForExpectNotVisible` and `assertTransactionAppeared` will throw an error if an error banner is seen or the transaction does not appear in the feed.